WeatherStormPower/BotConfigurationHandler.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Newtonsoft.Json;
+
+namespace WeatherStormPower;
+
+public class BotConfigurationHandler
+{
+    public static Dictionary<string, BotConfiguration> LoadConfiguration(string filePath)
+    {
+        string configurationJson = File.ReadAllText(filePath);
+        return JsonConvert.DeserializeObject<Dictionary<string, BotConfiguration>>(configurationJson);
```

Comment:
instead of deserializing to a dic of strings, do this:

```csharp
public class BotsConfiguration
{
    public BotConfiguration RainBot { get; set; }
    public BotConfiguration SunBot { get; set; }
    public BotConfiguration SnowBot { get; set; }
}
```

deserialize directly to `BotsConfiguration` instead of a dic.